packages/interface-ipfs-core/src/miscellaneous/id.js

```diff
@@ -44,5 +44,11 @@ module.exports = (common, options) => {
         expect(Multiaddr.isMultiaddr(ma)).to.be.true()
       }
     })
+
+    it('should have protocols property', async () => {
+      const res = await ipfs.id()
+
+      expect(res).to.have.a.property('protocols').that.is.an('array')
```

Comment:
This should assert that there are protocols present, at a minimum I think `/ipfs/id/1.0.0` should be supported

Reply:
There should be a handful of protocols, I'd assert them all as they shouldn't change very frequently. There may be differences in Node.js vs browser, depending on modules included, but I think they are currently equivalent.

Reply:
Got `["/floodsub/1.0.0", "/ipfs/bitswap/1.0.0", "/ipfs/bitswap/1.1.0", "/ipfs/bitswap/1.2.0", "/ipfs/id/1.0.0", "/ipfs/id/push/1.0.0", "/ipfs/ping/1.0.0", "/libp2p/circuit/relay/0.1.0", "/meshsub/1.0.0"]`, adding those to the assertions. Thanks guys!